contracts/libraries/GnosisAuction.sol

```diff
@@ -36,18 +36,6 @@ library GnosisAuction {
                 auctionDetails.gnosisEasyAuction
             );
 
-        if (
```

Comment:
why do we remove this?

Reply:
It's not needed because in most cases we're only approving the auction contract once, and we can trust that the auction will not overdraw the allowance